apps/dialtone-documentation/docs/.vuepress/baseComponents/AllTokens.vue

```diff
@@ -1,5 +1,43 @@
 <template>
-  <dt-stack :direction="{ default: 'row' }" gap="400">
+  <aside class="d-of-auto d-py32 lg:d-ps-relative lg:d-w100p d-ps-fixed dialtone-toc">
+    <h2 class="d-headline-eyebrow d-fw-semibold d-fc-secondary d-px12 d-pb4">
+      On this page
+    </h2>
+    <toc
+      :headers="filteredHeaders"
+      :options="tocOptions"
+    />
+  </aside>
+  <div class="tokens-bar">
+    <dt-input
+      id="search-input"
+      v-model="searchInput"
+      aria-label="Search tokens"
+      placeholder="Search Tokens / Value / Keyword"
+      type="text"
+      autocomplete="off"
+      @keyup="searchToken"
+    >
+      <template #leftIcon>
+        <dt-icon name="search" size="300" />
+      </template>
+      <template #rightIcon>
+        <dt-button
+          v-if="!isSearchEmpty"
```

Comment:
`isSearchEmpty` is being used once and using `!`

Could we call it sth like:
`isSearchNotEmpty`
`const isSearchNotEmpty = computed(() => searchInput.value && searchInput.value.trim().length > 0);`

Reply:
Maybe just `hasSearchTerm`. I usually try to avoid variable names with 'Not' in them because it's confusing.

Reply:
Agree, sure, it can be confusing sometimes.